content/_data/upgrades/2-277-1.adoc

```diff
@@ -0,0 +1,61 @@
+==== Configuration Form Modernization
```

Comment:
PR 4962 is not documented though it might require changes by users

Reply:
Thanks for catching that.  Will add [PR 4962](https://github.com/jenkinsci/jenkins/pull/4962) from the weekly changelog.

Reply:
Added 97206d871d5ccfc317b74fcf937dbe102eb3203c